cli/src/potpie/server_manager.py

```
@@ -0,0 +1,371 @@
+                        "--bind",
+                        "0.0.0.0:8001",
+                        "--log-level",
+                        "debug",
+                        "app.main:app",
+                    ],
+                    stdout=logout,
+                    stderr=logout,
+                )
+            
+                if(server_process.returncode != 0):
+                    raise StartServerError(f" Start Server Failed: {e}")
```

Ruff (0.8.2)

Undefined name `e`

(F821)